packages/sync-server/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "sync-server",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Server for syncing data between browser and server in Tupaia",
+  "homepage": "https://github.com/beyondessential/tupaia",
+  "bugs": {
+    "url": "https://github.com/beyondessential/tupaia/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://bes.au)",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "rm -rf dist && npm run --prefix ../../ package:build:ts",
+    "build-dev": "npm run build",
```

```suggestion
    "build-dev": "yarn run build",
```

(Don’t forget `packages/sync/package.json`!)